modules/ingest-common/src/main/java/org/elasticsearch/ingest/common/GrokProcessor.java

```diff
@@ -81,6 +96,10 @@ public Grok getGrok() {
         return grok;
     }
 
+    public boolean isIgnoreMissing() {
```

Comment:
the `isIgnoreMissing` getters are only used in tests, right? So then it can be package protected?